modules/upgrade.adoc

```diff
@@ -20,19 +20,23 @@ All Kubernetes objects and PVs in each {product-title} cluster are backed up as
 ====
 
 [id="upgrade-automatic_{context}"]
-== Automatic upgrades
+== Recurring upgrades
 
 Upgrades can be scheduled to occur automatically on a day and time specified by the cluster owner or administrator. Upgrades occur on a weekly basis, unless an upgrade is unavailable for that week.
 
+If you select recurring updates for your cluster, you must provide an administrator’s acknowledgment. {cluster-manager} does not start scheduled y-stream updates for minor versions without receiving an administrator’s acknowledgment. For information about administrator acknowledgment, see link:https://docs.openshift.com/dedicated/upgrading/osd-upgrading-cluster-prepare.html#upgrade-49-acknowledgement_osd-updating-cluster-prepare[Administrator acknowledgment when upgrading to OpenShift 4.9].
+
 [NOTE]
 ====
-Automatic upgrade policies are optional and if they are not set, the upgrade policies default to manual.
+Recurring upgrade policies are optional and if they are not set, the upgrade policies default to individual.
 ====
 
 [id="upgrade-manual_upgrades_{context}"]
-== Manual upgrades
+== Individual upgrades
+
+If you opt for individual upgrades, you are responsible for updating your cluster. If you select an update version that requires approval, you must provide an administrator’s acknowledgment. For information about administrator acknowledgment, see link:https://docs.openshift.com/dedicated/upgrading/osd-upgrading-cluster-prepare.html#upgrade-49-acknowledgement_osd-updating-cluster-prepare[Administrator acknowledgment when upgrading to OpenShift 4.9].
```

Comment:
Link?

Reply:
upgrade vs. update?
_If you opt for individual **_upgrades_**, 
you are responsible for **_updating_** your cluster_
 If you select an **_update_** version

Reply:
This requires a larger conversation with the PM team and the decision will impact more topics than the ones touched on in this PR. I'll create a separate PR to track the update vs upgrade term change, but meanwhile, this PR needs to be merged. Thank you.